.rubocop_todo.yml

```diff
@@ -104,20 +104,21 @@ Lint/NonDeterministicRequireOrder:
     - 'spec/request/v1/apidoc/apidoc_spec.rb'
     - 'spec/service_consumers/pact_helper.rb'
 
-# Offense count: 5
+# Offense count: 6
 # Cop supports --auto-correct.
 Lint/RedundantCopDisableDirective:
   Exclude:
     - 'app/models/emis_redis/military_information_v2.rb'
     - 'app/models/saved_claim/education_benefits/va_10203.rb'
     - 'app/policies/evss_policy.rb'
     - 'spec/jobs/facilities/pssg_download_spec.rb'
+    - 'spec/request/authentication/standard_authentication_spec.rb'
     - 'spec/support/saml/response_builder.rb'
 
 # Offense count: 1
 Lint/StructNewOverride:
   Exclude:
-    - 'spec/request/authentication/standard_authentication.rb'
+    - 'spec/request/authentication/standard_authentication_spec.rb'
```

Comment:
This line doesn't need to be here now. Right?

Reply:
This is a different rubocop error. RSpec/FilePath was failing when _spec.rb wasn't at the end of the filename